frontend/src/api/board.js

```
@@ -0,0 +1,226 @@
+    const response = await api.get(`/api/board/${boardId}/comment`)
+    return response.data
+  } catch (error) {
+    throw error
+  }
+}
+
+// 인기 게시글 조회 (RecommendationSection용)
+export const getPopularBoards = (successCallback, errorCallback) => {
+  getBoards({ sort: 'likeCount', direction: 'desc', size: 12 })
+    .then(response => {
+      console.log('Popular boards response:', response) // 디버깅용
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)